src/main/java/org/myrobotlab/service/GoogleSearch.java

```diff
@@ -229,7 +228,7 @@ public SearchResults search(String searchText) throws IOException {
     try {
       // can only grab first 100 results
 
-      String url = "https://www.google.com/search?lr=lang_" + locale.getTag() + "&site=imghp&tbm=isch&source=hp&q=" + searchText + "&gws_rd=cr";
+      String url = "https://www.google.com/search?lr=lang_" + Runtime.getInstance().getLocaleTag() + "&site=imghp&tbm=isch&source=hp&q=" + searchText + "&gws_rd=cr";

```

Comment:
this should not be from runtime.. this should be the locale of the google search service.   you should be able to search in french even if your current runtime locale is english.